.github/workflows/ci.yml

```diff
@@ -6,13 +6,44 @@ on:
 
 jobs:
     lint:
-        name: 'Lint changed files'
-        runs-on: 'ubuntu-latest'
+        name: 'Lint Changed Files'
+        runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
               with:
-                fetch-depth: 0
+                  fetch-depth: 0
             - uses: ./.github/actions/setup
             - uses: ./.github/actions/lint
               with:
                   WORKFLOW: 'ci'
+    test:
+        name: Build and Test
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+            - uses: ./.github/actions/setup
```

Comment:
are you reusing the cache? You create one with the lint action IIRC 🤔

Reply:
Yes, the cache is used on the pnpm install of each job

<img width="1226" alt="image" src="https://user-images.githubusercontent.com/35579930/195641833-629490a8-c043-49aa-be58-f7fa4b15cb9f.png">

https://github.com/coveo/plasma/actions/runs/3243534496/jobs/5318386560#step:3:61

Its not yet used for the build output, but that would be nice in a later step because the build takes a long time

Reply:
Still nice to have between steps already 😄